src/utils/utils.ts

```diff
@@ -541,11 +541,21 @@ export function getSourceFromLocations(
 export function runStarkNetClassHash(filePath: string): string {
   const warpVenvPrefix = `PATH=${path.resolve(__dirname, '..', '..', 'warp_venv', 'bin')}:$PATH`;
 
-  const classHash = execSync(`${warpVenvPrefix} starknet-class-hash ${filePath}`).toString().trim();
-  if (classHash === undefined) {
-    throw new Error(`starknet-class-hash failed`);
+  try {
+    return execSync(`${warpVenvPrefix} starknet-class-hash ${filePath}`).toString().trim();
+  } catch (e: any) {
```

Comment:
If you remove explicit `any` I think there are no linting errors

Reply:
by default is `unknown`, so you can't get any member access. When I tried to restrict to a more specific class it say that it's only two possible value: `any` and `undefined`

Reply:
So maybe use `Error` interface instead

Reply:
It won't work,
> Catch clause variable type annotation must be 'any' or 'unknown' if specified.ts(1196)
https://stackoverflow.com/questions/69021040/why-catch-clause-variable-type-annotation-must-be-any

Reply:
I see, honestly I'm not that against `any` if other solutions will prove to be cumbersome let's use eslint disable line just to silence the linter